treeherder_config.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+TREEHERDER_CONFIG = os.environ.get('TREEHERDER_CONFIG') or 'credentials.ignore'
+
+config = {
+    #"treeherder_url": "http://local.treeherder.mozilla.org",
+    "treeherder_url": "https://treeherder.allizom.org",
+
```

Comment:
Hard-coded? So to go to production, we have to change source code? Shouldn't both options be available and this be controlled by a switch {from an ini file|command-line argument|environment variable} so we can have one branch?